Neuron-PRDs/WalletConnect/GeneralComponent.md

```diff
@@ -44,3 +44,30 @@ To cancel this request, just press the Reject button.
 After the connection is established, the dapp will maintain a persistent connection with the wallet. The relevant connection information, the confirmed information from the previous step, will still be displayed here. Users can disconnect from the dapp at any time.
 
 ![ConnecttingInformation](pic/2023-05-30-08-41-25.png)
+
+### Signing Tx flowchart
+![picture 1](pic/967d539ec2b15727f459b9e33a70d0630de6a63f58e57e93ffe197de29380c01.png)  
+When dApp initializes a transaction, this component will pop up/ show.
+[UI for Signing Tx flowchart](https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=16536-38593)
+#### To confirm/reject a transaction from dapp
+
+- Normally, if the user want sign the transaction by simply click the NEXT btn then click CONFIRM btn.
+![picture 2](pic/f1d509997ae20bdb51032484bb33e5b704acdcef581fd53c7280b66747f5a54b.png)  
+
+- To reject or cancel this transaction , you can click REJECT Btn
+on the first step.
+
+
+#### To modify the transaction information
+
+- Click the pencil like icon
+![picture 3](pic/dde9d7d213ce52ce6b1f9febcfc2585cc743ff7ac0d0938cbcd64de595092660.png)  
+
+- Then modify information in corresponding text field, and click NEXT to continue.
+
+#### Further examine the receiving address and its detailed data
+
+- Click the arrow next to the address. The Data and Witness will display(elaborately speaking, these details will move upwards)
+![picture 4](pic/70d8140d96601991d8aa2632fc65d0c2a1eccef17ed02505fe50b9adade86115.png)  
+- In the UI demo, click this arrow, users can view all the receiving addresses, then view the data and witness the one you want by click the arrow next to the address on list.
```

Comment:
The detail should be described by text clearly and abstractly.
`click this arrow` is concrete, `check more addresses` is abstract

Reply:
How is the abbreviation of address list shown should be described.

In the screenshot, `To address: ckt...s49 / ckt...82 / ...` is meaning less.
What if show the addresses in various format based on the count:
1. only 1 address: display the full address
2. more than 1 address: show the count of addresses and check address list in another view.